index.js

```diff
@@ -10,6 +10,27 @@ const {
   READ_INDEX
 } = require('./lib/indexes')
 
+class FakeWeakRef {
```

Comment:
should we do this or should we just drop support?

Reply:
We need to support Node.js v12, it's still LTS and we do not want to force one of our consumers (Fastify) to drop it.